clang/test/Driver/sycl-offload-new-driver.c

```diff
@@ -119,3 +131,14 @@
 // RUN:   | FileCheck -check-prefix WRAPPER_OPTIONS_LINK %s
 // WRAPPER_OPTIONS_LINK: clang-linker-wrapper{{.*}} "--triple=spir64"
 // WRAPPER_OPTIONS_LINK-SAME: "--sycl-target-link-options={{.*}}-link-opt{{.*}}"
+
```

Comment:
This test will need to be extended to show the checks for AOT compilation calls, if/when that support is added.

Thanks